packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -44,6 +40,58 @@ const axiosRequestConfigToFetchRequestConfig = (
 	return [requestInfo, requestInit];
 };
 
+export interface IR11sResponse<T> {
+	content: T;
+	headers: Map<string, string>;
+	propsToLog: ITelemetryProperties;
+	requestUrl: string;
+}
+
+export function createR11sResponseFromContent<T>(content: T): IR11sResponse<T> {
+	return {
+		content,
+		headers: new Map(),
+		propsToLog: {},
+		requestUrl: "",
```

Comment:
should we pass requestUrl as argument here? It's weird to see empty string here.

Reply:
Added description as this is utility function to create response when we don't have any additional props here. We just have content.